assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -284,14 +284,17 @@ export class Fixture {
     // wait for the media gallery items to load, as many tests assume they're
     // there
     let mediaElements;
-    await waitFor(() => {
-      mediaElements = this.querySelectorAll('[data-testid^=mediaElement]');
-      if (!mediaElements?.length) {
-        throw new Error(
-          `Not ready: only found ${mediaElements?.length} media elements`
-        );
-      }
-    });
+    await waitFor(
+      () => {
+        mediaElements = this.querySelectorAll('[data-testid^=mediaElement]');
+        if (!mediaElements?.length) {
+          throw new Error(
+            `Not ready: only found ${mediaElements?.length} media elements`
+          );
+        }
+      },
+      { timeout: 5000 }
```

Comment:
So I added this to try to get karma tests to pass and it worked. Is this OK to commit? @swissspidy @BrittanyIRL?

Reply:
good question, definitely defaulting to @swissspidy  but hey it's nice to see green!

Reply:
At least we know that this helped 🤷‍♂️

Reply:
Works for me!